ui/pages/create-account/connect-hardware/index.js

```diff
@@ -288,7 +288,7 @@ class ConnectHardwareForm extends Component {
             this.state.device,
             // eslint-disable-next-line react/jsx-key
             <a
-              href="https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet"
+              href="https://metamask.zendesk.com/hc/en-us/articles/4408552261275"
```

Comment:
Can we make this url a constant in shared/constants/hardware-wallets?

Reply:
I did my best attempt at it! Let me know if this will work; I added the following to shared/constants/hardware-wallets:

/**
 * A commonly-used URL linking users to the Hardware Wallet Hub, the 'landing page' for info about HWWs in the Knowledge Base.
 */
export const HWW_HUB = {
  HUB_URL: 'https://metamask.zendesk.com/hc/en-us/articles/4408552261275'
}

Reply:
I offered a suggestion for a better naming convention, but beyond that you'll constant and use it in the place of these. If you'd prefer me to take over and make that change just lemme know, happy to help.